packages/ai-google/src/node/google-language-model.ts

```diff
@@ -127,7 +148,22 @@ export class GoogleModel implements LanguageModel {
     ) { }
 
     protected getSettings(request: LanguageModelRequest): Readonly<Record<string, unknown>> {
-        return request.settings ?? {};
+        const settings = request.settings ?? {};
+        // Filter out RetrySettings keys to avoid passing them to the API
+        const apiSettings = { ...settings };
+        delete apiSettings.maxRetriesOnErrors;
```

Comment:
we usually pass these kind of settings using the constructor.
The advantage is that you can expose these settings via a preference (which I think should be done). This allows the user to influence the settings. Currently you have to know the syntax and add these model unrelated settings to the modelsettings object.

Reply:
Ok. Thanks. 
I have changed the settings into preferences and updated the PR description accordingly.